auth/src/commonMain/kotlin/com/liftric/auth/jwt/CognitoAccessToken.kt

```diff
@@ -12,7 +12,9 @@ data class CognitoAccessTokenClaims(
     override val exp: Long,
     override val iat: Long,
     override val iss: String,
-    override val jti: String,
+    override val jti: String? = null,
+    @SerialName("origin_jti")
```

Comment:
Linked to the comment below: I can feel the need for `@SerialName` here but wondered regarding the use below. I just asked myself why we don't have to custom serialiser or none? 😏

Reply:
The CognitoIdToken needs a custom serializer because of possible custom attributes. The CognitoAccessToken won't have custom attributes (afaik), thus the standard serializer and use of @SerialName.